### PR TITLE
[release/2.2] Change prerelease label to be more accurate

### PR DESCRIPTION
**Pull request by mmitche**
Not critical, but better than servicing

### CODE REVIEW

File: dependencies.props [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
     <StableVersion Condition="'$(StabilizePackageVersion)' == 'true' and '$(StableVersion)' == ''">$(PackageVersion)</StableVersion>
 
-    <PreReleaseLabel>servicing</PreReleaseLabel>
+    <PreReleaseLabel>preview</PreReleaseLabel>
```
Comment (weshaggard) [inline dependencies.props]: should it not be preview1?

Comment (weshaggard) [reply dependencies.props]: or preview2 now?

Comment (mmitche) [reply dependencies.props]: Actually, preview2.

**Comment by mmitche** [comment]
@dotnet-bot test this please

File: dependencies.props (hunk 1 above)
Comment (weshaggard) [reply dependencies.props]: You plan to do the same in the other repo's?

**Pull request merged**